version-info.toml

```diff
@@ -10,6 +10,11 @@ channel = "nixos-25.05"
 status = "stable"
 
 [versions."25.11"]
+branch = "nixos-25.11"
+channel = "nixos-25.11"
+status = "beta"
```

Comment:
Merging this without first creating the 25.11 branch will cause things like the docs workflow to fail.

We should probably do the branch off now.

Reply:
From the current `main` branch or from this PR?